internal/cmd/connect/command_plugin_install.go

```diff
@@ -279,6 +282,30 @@ func getPluginDirFromFlag(cmd *cobra.Command) (string, error) {
 	return pluginDir, nil
 }
 
+func getWorkerConfigsFromFlag(cmd *cobra.Command) ([]string, error) {
+	if !cmd.Flags().Changed("worker-configs") {
+		return nil, nil
+	}
```

Comment:
This doesn't look like it's needed. If the flag isn't passed, we'll default to an empty slice, and nothing will happen below.